internal/cephfs/controllerserver.go

```diff
@@ -134,6 +135,9 @@ func (cs *ControllerServer) createBackingVolumeFromSnapshotSource(
 	})
 	if err != nil {
 		log.ErrorLog(ctx, "failed to create clone from snapshot %s: %v", sID.FsSnapshotName, err)
+		if errors.Is(err, syscall.EAGAIN) {
```

Comment:
Checking for the syscall error is not very clean, and is potentially not correct. It would be much better to check for a valid go-ceph error.

Can you explain how you verified that checking against `syscall.EAGAIN` is correct?

Reply:
Currently go-ceph doesn't have a compatible error code for EAGAIN to check for. Since we get EAGAIN from ceph with the proposed change and that will be formated by go-ceph and sent back to csi, checking it against `syscall.EAGAIN` should work for the time being.

I will add a `TODO` note for implementing EAGAIN error handle in go-ceph and use that once available.